tutorcelery/patches/openedx-cms-production-settings

```diff
@@ -2,3 +2,6 @@ try:
     EXPLICIT_QUEUES.update({{CELERY_CMS_EXPLICIT_QUEUES}})
 except NameError:
     EXPLICIT_QUEUES = {{CELERY_CMS_EXPLICIT_QUEUES}}
+
+# Prevents losing tasks when workers are shutdown
+CELERY_ACKS_LATE = True
```

Comment:
Should we keep the [prefetch multiplier](https://github.com/eduNEXT/drydock/pull/129/files#diff-910f8be9117d213f09354a70413dcb77bb950f3cc603cc8810ead0e70c7232edL19) setting?

Reply:
Based on load tests we know that value reduces a lot the performance with many celery tasks. Not sure if we should keep it.

Reply:
According to the [celery docs](https://docs.celeryq.dev/en/stable/userguide/optimizing.html#reserve-one-task-at-a-time) this is suitable for workers with long-running tasks. It is not a priority to use it right now I think